python/podio_schema_evolution.py

```diff
@@ -469,4 +475,4 @@ def read_evolution_file(self) -> None:
     comparator.read()
     comparator.compare()
     comparator.print_comparison()
-    print(comparator.get_changed_schemata(schema_filter=root_filter))
+    #print(comparator.get_changed_schemata(schema_filter=root_filter))
```

Comment:
I suppose this was some debug output at some point, which is why it is removed?

Reply:
sort of. still useful for the future to include again